### ISSUE TITLE
[REQUEST] Alt-Tab Icons Close Window (X) Buttons

### ISSUE THREAD

**Issue by cebroski** [linuxmint/cinnamon#10719]
```
Linux Mint 20.3, 64-bit,  Cinnamon 5.2.7

Linux Kernal = 5.4.0-104-generic

Graphics = Advanced Micro Devices, Inc. [AMD/ATI] Cedar [Radeon HD 5000/6000/7350/8350 Series] (prog-if 00 [VGA controller])

 ```

**Issue**
Alt-tab icons are missing a close window (X) button. This would improve work flow for former Windows/ Mac users.

**Steps to reproduce**
Press Alt+Tab and release the tab key while holding down on the alt key. Hover the mouse arrow over the icons of the alt-tab open windows preview in the center of the screen and notice that there is no close window button.

**Expected behavior**
I expect there to be a close Window button like in Windows and Mac OS.

**Other information**
Windows and Mac OS has this. Linux Mint should too!! :)

Thanks for reading and have a blessed day!

**Comment by smurphos** [comment]
Have you tried the Scale view (Ctrl-Alt-Down is the default keyboard shortcut - but can also be accessed via hotcorner, or panel applet). Full screen overview of open windows with close button present and correct.

**Comment by Secret-chest** [comment]
> Have you tried the Scale view (Ctrl-Alt-Down is the default keyboard shortcut - but can also be accessed via hotcorner, or panel applet). Full screen overview of open windows with close button present and correct.

Yes it's better for that sort of thing.

**Comment by cebroski** [comment]
Yes, I have but I would also like to have another way to close windows with
Alt-Tab. Thank you!! :)
